src/clevertap.js

```
@@ -973,6 +979,22 @@ export default class CleverTap {
   }
 
+  getVariables () {
+    return reconstructNestedObject(
+      StorageManager.readFromLSorCookie(VARIABLES)
+    )
+  }
+
+  getVariableValue (variableName) {
+    const variables = StorageManager.readFromLSorCookie(VARIABLES)
+    const reconstructedVariables = reconstructNestedObject(variables)
+    if (variables.hasOwnProperty(variableName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)